src/aqua/hex.aqua

```diff
@@ -0,0 +1,26 @@
+module Hex declares inc, dec, min
+
+import FluenceAuroraConnector from "services.aqua"
+import SpellId from "../types.aqua"
+import spell_log from "../fluence/spell.aqua"
```

Comment:
```suggestion
import SpellId from "./types.aqua"
import spell_log from "./fluence/spell.aqua"
```

Otherwise, it's not compiled for me

I wonder, why do the tests pass

Reply:
Oh, I guess it's because of the command
```bash
fluence aqua --import src/aqua --import src/aqua/decider --import src/aqua/chain --import src/aqua/fluence -i src/aqua/decider/deal_spell.aqua -o "$DISTRO_DIR/deal_spell" --air
```
This one generates errors
```bash
fluence aqua -i src/aqua/hex.aqua --air -o x
```

Reply:
Yeah, there are some bugs in there still